Dockerfile.alpine

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.6.3-alpine3.9
+
+RUN apk --no-cache add curl nodejs nodejs-npm postgresql-client postgresql-dev fftw fftw-dev
```

Comment:
I think we may want to add `apk --no-cache upgrade && ..` at the beginning, make sure we get any security updates not in the base image

Reply:
👍